### PR TITLE
Update file number validation

### PR DESCRIPTION
**Pull request by jbalboni**
Pulls in the updated schema to get the validation pattern change and updates the message based on Peggy's feedback.

screen shot 2017-09-20 at 2 08 15 pm

### CODE REVIEW

**Review by cehsu — APPROVED** [review]
LGTM.

File: src/js/burials/config/form.js [hunk 1]
```diff
@@ -162,7 +162,7 @@ const formConfig = {
                 widgetClassNames: 'usa-input-medium'
               },
               'ui:errorMessages': {
-                pattern: 'File number must be 8 digits'
+                pattern: 'Your VA file number must be between 7 to 9 digits.'
```
Comment (cehsu) [inline src/js/burials/config/form.js]: Are periods common at the end of validation messages? Just asking as it looks like the previous message didn't have one.

Comment (jbalboni) [reply src/js/burials/config/form.js]: @peggygannon asked for it, but no, I don't think most of our messages have them.

Comment (cehsu) [reply src/js/burials/config/form.js]: @peggygannon should we be adding periods to the end of validation messages of a certain length (or even all of them)?

Comment (peggygannon) [reply src/js/burials/config/form.js]: @cehsu I had originally indicated to add a period, but if previous validation messages don't have them, then I'm OK with keeping that style.

Comment (jbalboni) [reply src/js/burials/config/form.js]: I will go look through the ones we have to be sure.

Comment (peggygannon) [reply src/js/burials/config/form.js]: @cehsu @jbalboni one more thought... it's be great to audit our validation messages to see if they have/don't have periods.

Comment (jbalboni) [reply src/js/burials/config/form.js]: I took a quick look, I didn't see an instance of one with a period.

Comment (peggygannon) [reply src/js/burials/config/form.js]: @jbalboni OK, great. Thanks for checking!

**Pull request merged**